src/feed-leaf-container.tsx

```diff
@@ -28,6 +27,16 @@ export class Container extends React.Component<Properties> {
     };
   }
 
+  getTwitterLink = (): string => {
```

Comment:
is there a reason that this is in the container, and not just in the component where it is being used?

this feels like something that is the responsibility of the thing that is actually rendering it, and not something that needs to be passed as a function. can we just add this to the FeedLeaf itself, and then just test that the correct href is being given to the link, based on the znsRoute that is passed in?

eg. one test could be:
```
  it('renders link to twitter', () => {
    const wrapper = subject({ znsRoute: 'wilder.WoW.poster.001' });

    const expectedLink = 'https://twitter.com/intent/tweet?url=https://share.market.wilderworld.com/WoW.poster.001';

    expect(wrapper.find('.feed-leaf__share').prop('href')).toStrictEqual(expectedLink);
  });
  ```